chrome/content/zotero/xpcom/citeprocRsBridge.js

```diff
@@ -0,0 +1,292 @@
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	Copyright © 2019 Center for History and New Media
+					George Mason University, Fairfax, Virginia, USA
+					http://zotero.org
+	
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+(function() {
+
+const additionalCSLProperties = ['locator', 'label', 'suppress-author', 'author-only', 'prefix', 'suffix'];
+
+/**
+ * Bridging code which allows to use citeproc-js and citeproc-rs almost interchangeably
+ * without big alterations to existing Zotero code
+ */
+Zotero.CiteprocRs = {
```

Comment:
Style: `CiteprocRs` → `CiteprocRS`, everywhere

Reply:
We can't (simply) do this as the special Zotero build from CiteprocRs exports its classes onto the `Zotero.CiteprocRs` object and that's part of a [v0.1.0](https://github.com/zotero/citeproc-rs/releases/tag/wasm-v0.1.0) npm build.

Reply:
Sad